QUEST.tsv

```diff
@@ -1214,24 +1214,24 @@ QUEST_20150317_001214	Modestas knew the these gambles were more than just about
+QUEST_20150317_001226	Everyone who were there on the spot favored the idea. {nl} The Alchemist took the gold he obtained from Modestas and paid it to the Dievdirbys as commission, and went to get permission from the Knight Leader and Bishop of Klaipeda.{nl}
+QUEST_20150317_001227	As news spread, everyone in Klaipeda was pleased to know a Goddess Statue was planned for their city. {nl} However, one last problem remained aside from winning their favor. And that question was, which Goddess should the statue commemorate? {nl} The soldiers who gave help transporting the boulder were especially passionate about this issue, and began quarrelling. {nl}
+QUEST_20150317_001228	The one who answered this question was none other than Modestas. {nl} Without any real intent, Modestas argued that the Goddesses in power have guided him via divine providence to build a statue at Klaipeda that day. {nl}
+QUEST_20150317_001229	{memo X}That the old lady appeared before him, that the many bets occurred, that the mountain erupted a stone for material was the divine providence of the Goddesses, argued Modestas, persuading everyone. {nl} Thus, the people decided to follow the words of Modestas, and Modestas relayed to the Dievdirbys the order to create the statue. {nl} When the statue was finally finished, the citizens of Klaipeda gathered, curious to know the identity of the Goddess who had been raised in the center of their beloved city.{nl}
+QUEST_20150317_001230	However, to their bewilderment, not a single soul could recognize the Goddess whom the completed statue commemorated.{nl}
+QUEST_20150317_001231	{memo X}The Dievdirbys who had sculpted the statue had already vanished deep into the Kateen Forest, so the citizens of Klaipeda repeatedly questioned Modestas for her name. {nl} Despite the numerous questions nobody received an answer from Modestas. {nl}At any rate, this is the story of how the Goddess Statue of Klaipeda came to be. Even now that statue stands in the center of Klaipeda.{nl}
+QUEST_20150317_001232	And so, some people believed Modestas, having undertaken the Goddesses' intervention, was an honest man who did not gamble. In reality, Modestas didn't stop making gambles until his death. {nl}
+QUEST_20150317_001233	{memo X}Since that time, whenever Modestas took on a wager, he always betted the same thing. The answer to the question of who the Statue of Klaipeda honored.{nl}To uncover the name of the Goddess, numerous people challenged Modestas to a bet.{nl} {nl} But no matter the impossible odds, Modestas never lost a bet. Therefore nobody was able to win the answer from Modestas for years and time flew by.{nl}
+QUEST_20150317_001234	Though Modestas died an undefeated, invincible gambler, by then he had as mentioned never bet anything other than the Goddess' identity, living purely off the profits gained from such bets through the end of his life. {nl}
```

Comment:
Agree, the story seemed to fall in place really quickly, and a lot of small details were added that only thicken the language.